config/locales/en.yml

```diff
@@ -2786,6 +2787,21 @@ en:
           primary_disabled_question: Are you disabled according to the New Jersey definition?
           spouse_disabled_question: Is your spouse disabled according to the New Jersey definition?
           title: Disabled Exemption
+      nj_eligibility_health_insurance:
+        edit:
+          label: Did you and all members of your tax household have health insurance (with minimum essential coverage) for every month in 2024?
+          title: Let’s get started.
+          helper_description_html: |
+            <p>Most people who have health insurance have minimum essential coverage. “Minimal essential coverage” just refers to how much your plan covers. If you have any of the following, you most likely have it:</p>
+            <ul>
+              <li>A plan through your employer</li>
+              <li>A plan bought through <a href="https://nj.gov/getcoverednj/">GetCoveredNJ</a></li>
```

Comment:
[pebble] these external links need rel="noopener noreferrer" and we may also want to do target="_blank" since being taken out of a workflow is pretty annoying